testsuite/tests/typing-modules/inclusion_errors.ml

```diff
@@ -1853,3 +1853,183 @@ Error: Signature mismatch:
        The value "x" is required but not provided
        The value "y" is required but not provided
 |}];;
+
+
+module Eq_label: sig
+  type t = int -> int
+end = struct
+  type t = x:int -> int
+end
+[%%expect {|
+Lines 3-5, characters 6-3:
+3 | ......struct
+4 |   type t = x:int -> int
+5 | end
+Error: Signature mismatch:
+       Modules do not match:
+         sig type t = x:int -> int end
+       is not included in
+         sig type t = int -> int end
+       Type declarations do not match:
+         type t = x:int -> int
+       is not included in
+         type t = int -> int
+       The type "x:int -> int" is not equal to the type "int -> int"
+       The argument "x" is labeled, but an unlabeled argument was expected
```

Comment:
Proposal: `The argument has label "x", but ...`.

Reply:
Counterproposal: `The first argument is labeled "x"`

Reply:
Fine with me.